### PR TITLE
Bind ports that must be externally accessable to all interfaces

### PR DESCRIPTION
**Pull request by reinkrul**

### CODE REVIEW

File: single/docker-compose.yml [hunk 1]
```diff
@@ -4,9 +4,10 @@ services:
     image: nutsfoundation/nuts-node:master
     environment:
       NUTS_CONFIGFILE: /opt/nuts/nuts.yaml
+      NUTS_NETWORK_BOOTSTRAPNODES: ""
     ports:
       - "127.0.0.1:1323:1323"
```
Comment (woutslakhorst) [inline single/docker-compose.yml:9]: this one as well for `/n2n` and the access token

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: single/docker-compose.yml [hunk 2]
```diff
@@ -4,9 +4,10 @@ services:
     image: nutsfoundation/nuts-node:master
     environment:
       NUTS_CONFIGFILE: /opt/nuts/nuts.yaml
+      NUTS_NETWORK_BOOTSTRAPNODES: ""
```
Comment (woutslakhorst) [inline single/docker-compose.yml:7]: in the workshop, we'll change the config file. This will override that?

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: single/docker-compose.yml (hunk 2 above)
Comment (reinkrul) [reply single/docker-compose.yml:7]: Ah I assumed that in the workshop the docker-compose file would be edited, so I added this as convenience for the participants, so they can't make mistakes on how to spell/underscore it. But in that case I'll remove it.

File: single/docker-compose.yml (hunk 1 above)
Comment (reinkrul) [reply single/docker-compose.yml:9]: Ouch, that means all administrative operations are exposed as well... Ofc not a possibility for a "data leak" with production data, but maybe a bit sloppy?

We could bind `/n2n` to a different port...?

Comment (woutslakhorst) [reply single/docker-compose.yml:8]: Doesn't really matter for demo purposes. In production your reverse proxy will require client certs for that interface/path.

**Pull request merged**